### ISSUE TITLE
output message color incorrect when calling from Jenkins shell

### ISSUE THREAD

**Issue by dourgulf** [supermarin/xcpretty#308]
The Jenkins log window do not display any colorful compile error message.

**Comment by ftrofin** [comment]
Did you try adding `--color` to the xcpretty argument list? i.e.
```
 blah | xcpretty --color
```
I think autodetect doesn't always work.

**Comment by dourgulf** [comment]
No, Log message will be display with "[33m"  ahead.

**Comment by dourgulf** [comment]
Yes, add --color and turn on Jenkins ANSI color plugin, it show the correct color. 
Many thanks!
